services/src/layers/storage.rs

```diff
@@ -19,7 +21,7 @@ pub enum LayerDbError {
 }
 
 #[async_trait]
-pub trait LayerDb: Send + Sync {
+pub trait LayerDb: LayerCollectionProvider + Send + Sync {
```

Comment:
Why is it good to move the two methods into a separate trait?
Isn't this trait implemented for every `LayerDb` and solely for `LayerDb`s?

Reply:
no, the LayerDB stores layers (inside Geo Engine). The LayerCollectionProviders provides access to layers. And they can be stored inside Geo Engine or be server from an external provider.